android/src/main/java/com/twiliovoicereactnative/NotificationUtility.java

```diff
@@ -111,26 +106,24 @@ public static Notification createCallAnsweredNotificationWithLowImportance(@NonN
       callRecord.getUuid());
     PendingIntent piEndCallIntent = constructPendingIntentForReceiver(context, endCallIntent);
 
-    RemoteViews remoteViews = new RemoteViews(context.getPackageName(), R.layout.custom_call_in_progress);
-    remoteViews.setTextViewText(R.id.make_call_text, getContentBanner(context));
-    remoteViews.setOnClickPendingIntent(R.id.end_call, piEndCallIntent);
-
     return constructNotificationBuilder(context, Constants.VOICE_CHANNEL_LOW_IMPORTANCE)
       .setSmallIcon(smallIconResId)
-      .setContentTitle(title)
-      .setContentText(getContentBanner(context))
       .setCategory(Notification.CATEGORY_CALL)
       .setAutoCancel(false)
-      .setCustomContentView(remoteViews)
-      .setCustomBigContentView(remoteViews)
-      .setContentIntent(pendingIntent)
-      .setFullScreenIntent(pendingIntent, true)
+      .setContentIntent(piForegroundIntent)
+      .setFullScreenIntent(piForegroundIntent, true)
+      .setOngoing(true)
+      .addPerson(activeCaller)
+      .setStyle(NotificationCompat.CallStyle.forOngoingCall(activeCaller, piEndCallIntent))
       .build();
   }
 
   public static Notification createOutgoingCallNotificationWithLowImportance(@NonNull Context context,
                                                                              @NonNull final CallRecord callRecord) {
     final int smallIconResId = getSmallIconResource(context);
+    final Person activeCaller =  new Person.Builder()
+      .setName("outgoing call")
```

Comment:
Can we expose this to the JS layer when the JS layer tries to make a call? Is there any utility to that?

Reply:
not with this pr..  when we discus allowing for users to provide custom  notifications, we can revisit it.